spateo/plotting/static/pairwise.py

```diff
@@ -0,0 +1,185 @@
+"""
+Plots for pairwise comparison analyses between groups of samples and across different conditions (e.g. biological
+conditions or using different clustering parameters).
+
+These functions build on the Label class- see spateo.utils.Label for documentation
+"""
+from typing import List, Tuple, Union
+
+import matplotlib as mpl
+import matplotlib.patheffects as PathEffects
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy
+from anndata import AnnData
+from matplotlib.collections import PolyCollection
+from matplotlib.ticker import StrMethodFormatter
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from ...utils import Label, interlabel_connections
+
+plt.rcParams["figure.dpi"] = 300
+
+
+def plot_connections(
```

Comment:
I realized that we need to add a legend for the meaning of the box size for this function

Reply:
<img width="497" alt="Screen Shot 2022-08-28 at 1 22 20 PM" src="https://user-images.githubusercontent.com/7456281/187086681-01cae659-78d6-4f22-8a34-db9a19a15f0b.png">
The x-axis ticks need to be on the top

Reply:
the ticks on the left axis is good